packages/shellcheck/build.sh

```
@@ -1,7 +1,13 @@
 #!/bin/sh
 set -ex
 
-tar -xof shellcheck-v${MINIMAL_ARG_VERSION}.linux.x86_64.tar.xz
+case $(uname -m) in
+  x86_64)  PLATFORM="x86_64" ;;
+  aarch64) PLATFORM="aarch64" ;;
+  *)       echo "unsupported architecture: $(uname -m)" >&2; exit 1 ;;
+esac
+
+tar -xof shellcheck-v${MINIMAL_ARG_VERSION}.linux.${PLATFORM}.tar.xz
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)